gradle.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=5.0.1
+projectVersion=5.0.2
```

Comment:
if you adding some new functionality you can bump MINOR version. in this case to 5.1.0 . Last number is PATCH number, for fixes etc.. more info about semantic versioning here https://semver.org/ .